changes/2020-06-17_encrypt-max-plaintext-length-input/change.md

```diff
@@ -0,0 +1,203 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Encrypt Max Plaintext Length Input
+
+## Affected Features
+
+| Feature                                           |
+| ------------------------------------------------- |
+| [Encrypt](../../client-apis/encrypt.md)           |
+| [CMM Interface](../../framework/cmm-interface.md) |
+
+## Affected Specifications
+
+| Specification                                     |
+| ------------------------------------------------- |
+| [Encrypt](../../client-apis/encrypt.md)           |
+| [CMM Interface](../../framework/cmm-interface.md) |
+
+## Affected Implementations
+
+| Language | Repository                                                                    |
+| -------- | ----------------------------------------------------------------------------- |
+| Python   | [aws-encryption-sdk-python](https://github.com/aws/aws-encryption-sdk-python) |
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+### Known-Length Plaintext
+
+Any plaintext input to the Encrypt operation where the total length of that plaintext
+can be immediately determined.
+For example, if a user supplies plaintext as a string or byte array to Encrypt,
+that is a known-length plaintext.
+
+### Unknown-Length Plaintext
+
+Any plaintext input to the Encrypt operation where the total length of that plaintext
+cannot be immediately determined.
+For example, if a user supplies plaintext as a stream to Encrypt,
+that is a unknown-length plaintext.
+
+## Summary
+
+The Encrypt operation takes a plaintext as input that can either have a known length or an
+unknown length.
+If the input plaintext is a unknown-length plaintext,
+the Encrypt operation MUST also take an optional input that bounds the length of that input.
+
+The specification already specifies a [plaintext length](../../client-apis/encrypt.md#plaintext-length)
```

Comment:
Should this be an absolute link (with the SHA)? So a different header in the future still works for the link?